src/utils/vcs.rs

```diff
@@ -915,6 +915,7 @@ fn git_create_commit(dir: &Path, file_path: &str, content: &[u8], commit_message
             "--author",
             "John Doe <john.doe@example.com>",
             "--quiet",
+            "--no-edit",
```

Comment:
For me, its rather `utils::vcs::test_find_matching_rev_with_lightweight_tag` that opens a vim window and then fails because `fatal: no tag message?`.

Maybe my local git is configured to always expect a message for tags, not sure

Reply:
I have this in my global git config:

```
[commit]
    gpgsign = true
[tag]
    gpgsign = true
```

Looks like you can only sign tags if they have a message.
Maybe an explicit command line switch to avoid signing them, or provide a default message (aka `-m=$version`) would solve this.